certifi/core.py

```diff
@@ -8,8 +8,18 @@
 """
 import os
 
+try:
+    from importlib.resources import read_text
+except ImportError:
+    def read_text(_module, _path, encoding="ascii"):
```

Comment:
I think there's an importlib backport for older versions much like `mock` is maintained now as a backport of `unittest.mock`.

@Lukasa @alex thoughts on having a conditional dependency on that backport so that this works consistently in use-cases like PyOxidizer? (I don't think it's unique in the fact that using `where` doesn't work well.)

Reply:
I don't have an opinion.

I would like a comment describing in what condition this fallback is hit, so we know when this code can be deleted.

Reply:
Hmm. I’m inclined not to bother: conditional dependency is a bit annoying. I would like @alex’s suggestion of when this fallback is hit to be added though.

Reply:
Done.